### ISSUE TITLE
Dockerfile output for CKV_DOCKER_7 and CKV_DOCKER_2 messed up

### ISSUE THREAD

**Issue by ac-hibbert** [bridgecrewio/checkov#1389]
**Describe the bug**

Seems like results of check CKV_DOCKER_2 should be showing for CKV_DOCKER_7

```
Check: CKV_DOCKER_7: "Ensure the base image uses a non latest version tag"
	FAILED for resource: /Dockerfile.FROM
	File: /Dockerfile:0-0

		0 | USER actions


Check: CKV_DOCKER_2: "Ensure that HEALTHCHECK instructions have been added to container images "
	FAILED for resource: /Dockerfile.
	File: /Dockerfile:1-1
	Guide: https://docs.bridgecrew.io/docs/ensure-that-healthcheck-instructions-have-been-added-to-container-images

		1 | FROM 290244732740.dkr.ecr.eu-west-1.amazonaws.com/actions:latest
```

**To Reproduce**
1. Run checkov for a fairly standard Dockerfile using an image with the latest tag and no healthcheck

**Expected behavior**
Checks show the correct output

**Screenshots**

**Desktop (please complete the following information):**
 - OS: macOS
 - Checkov Version: 2.0.275

**Additional context**
Add any other context about the problem here (e.g. code snippets).

**Comment by schosterbarak** [comment]
@ac-hibbert would you mind sharing a sample dockerfile?

**Comment by ac-hibbert** [comment]
Example Dockerfile

```
FROM jenkins:latest

USER root

RUN yum update -y && \
    yum install -y curl && \
    yum clean all

USER jenkins
```

Results:

```
$ checkov -f Dockerfile


       _               _
   ___| |__   ___  ___| | _______   __
  / __| '_ \ / _ \/ __| |/ / _ \ \ / /
 | (__| | | |  __/ (__|   < (_) \ V /
  \___|_| |_|\___|\___|_|\_\___/ \_/

By bridgecrew.io | version: 2.0.255
Update available 2.0.255 -> 2.0.318
Run pip3 install -U checkov to update


dockerfile scan results:

Passed checks: 3, Failed checks: 2, Skipped checks: 0

Check: CKV_DOCKER_5: "Ensure update instructions are not use alone in the Dockerfile"
	PASSED for resource: Dockerfile.
	File: Dockerfile:1-1
	Guide: https://docs.bridgecrew.io/docs/ensure-update-instructions-are-not-used-alone-in-the-dockerfile

Check: CKV_DOCKER_8: "Ensure the last USER is not root"
	PASSED for resource: Dockerfile.USER
	File: Dockerfile:8-8

Check: CKV_DOCKER_3: "Ensure that a user for the container has been created"
	PASSED for resource: Dockerfile.USER
	File: Dockerfile:2-2
	Guide: https://docs.bridgecrew.io/docs/ensure-that-a-user-for-the-container-has-been-created

Check: CKV_DOCKER_7: "Ensure the base image uses a non latest version tag"
	FAILED for resource: Dockerfile.FROM
	File: Dockerfile:0-0

		0 | USER jenkins


Check: CKV_DOCKER_2: "Ensure that HEALTHCHECK instructions have been added to container images "
	FAILED for resource: Dockerfile.
	File: Dockerfile:1-1
	Guide: https://docs.bridgecrew.io/docs/ensure-that-healthcheck-instructions-have-been-added-to-container-images

		1 | FROM jenkins:latest

```

**Comment by nwokou01** [comment]
Having same issues, I could see though that this seems to be more related with the results not pointing to the correct line with the identifies error,  CKV_DOCKER_2 and CKV_DOCKER_3 all seem to be pointing to the line for  CKV_DOCKER_7:

Version: 2.0.350 

Sample result with this error:

Check: CKV_DOCKER_7: "Ensure the base image uses a non latest version tag"
        PASSED for resource: Dockerfile.
        File: Dockerfile:1-1
        Guide: https://docs.bridgecrew.io/docs/ensure-the-base-image-uses-a-non-latest-version-tag

Check: CKV_DOCKER_2: "Ensure that HEALTHCHECK instructions have been added to container images "
        FAILED for resource: Dockerfile.
        File: Dockerfile:1-1
        Guide: https://docs.bridgecrew.io/docs/ensure-that-healthcheck-instructions-have-been-added-to-container-images

                1 | FROM python:3.6-slim
Check: CKV_DOCKER_3: "Ensure that a user for the container has been created"
        FAILED for resource: Dockerfile.
        File: Dockerfile:1-1
        Guide: https://docs.bridgecrew.io/docs/ensure-that-a-user-for-the-container-has-been-created

                1 | python:3.6-slim

**Comment by gruebel** [comment]
thanks to @Xitric this should be now fixed since version `2.0.472`.
